daprdocs/content/en/operations/configuration/use-custom-certificates.md

```diff
@@ -0,0 +1,93 @@
+---
+type: docs
+title: "How-To: Use custom certificates in the Dapr sidecar"
+linkTitle: "Use custom certificates"
+weight: 6500
+description: "Configure the Dapr sidecar container to trust custom certificates"
+---
+
+The Dapr sidecar can be configured to trust custom certificates for communicating with external services. This is useful in scenarios where a self-signed certificate needs to be trusted, e.g., using an HTTP binding, configuring an outbound proxy for the sidecar, etc. Both custom CA certificates and leaf certificates are supported.
+
+{{< tabs Self-hosted Kubernetes >}}
+
+{{% codetab %}}
+
+When the sidecar is not running inside a container, certificates must be directly installed on the host operating system. 
+
+When the sidecar is running as a container:
+1. Certificates must be available to the sidecar container. This can be configured using volume mounts.
+1. The environment variable `SSL_CERT_DIR` must be set in the sidecar container, pointing to the directory containing the certificates.
+1. For Windows containers only, the container needs to run with administrator privileges to be able to install the certificates.
+
+Below is an example that uses Docker Compose:
+```yaml
+version: '3'
+services:
+  dapr-sidecar:
+    image: "daprio/daprd:edge"
```

Comment:
add comment for making users aware that they need to use versions 1.8 or above ....

Reply:
So this doc is a part of v1.8, does not that make it implicit? Anything that's present in doc version X and not X-1, will need a min version X.

Reply:
Anyway I can add a tiny comment to be safe.